### DISCUSSION TITLE
请问主题文档基于什么制作（不是很重要的问题）

### DISCUSSION THREAD

**Discussion by Sunwuyuan** [YunYouJun/hexo-theme-starry#184] [Q&A 疑难杂症] [answered]
请问主题文档https://yun.yunyoujun.cn/是怎么做的，我发现好多主题文档都长这样........比如一言https://i.loli.net/2021/10/10/mquEHW7rxKLdtG3.png，某个主题？还是什么........

**Comment by Sunwuyuan** [comment]
GitHub自动超链接好像出问题了，上面不是一整个超链接

**Comment by YunYouJun** [comment, marked as answer]
- [Vuepress](https://vuepress.vuejs.org/)
- [Vitepress](https://vitepress.vuejs.org/): VuePress' spiritual successor

**Reply by Sunwuyuan** [reply]
谢谢😊
